### PR TITLE
[WIP] Widgetastic conversion of Settings Page

### PR DESCRIPTION
**Pull request by niyazRedhat**

### CODE REVIEW

File: cfme/base/ui.py [hunk 1]
```diff
@@ -329,6 +329,24 @@ def step(self):
             self.prerequisite_view.navigation.select('Settings', 'My Settings')
 
 
+class MySettingsView(BaseLoggedInPage):
+    @View.nested
+    class visualAll(Tab):  # noqa
```
Comment (mfalesni) [inline cfme/base/ui.py]: Use the snake_case in the nested classes names.

Comment (niyazRedhat) [inline cfme/base/ui.py]: @mfalesni ok sure. thanks

**Comment by dajoRH** [comment]
Lint report for commit cdef040566e08f7a5f779b1b946ff7cd2aa11ed8:

`cfme/configure/settings.py`:
- :bangbang: Line 45:1: **E302** *expected 2 blank lines, found 1*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request closed without merging**